examples/composite_dialog/my_app_window/mod.rs

```diff
@@ -0,0 +1,80 @@
+mod imp;
+
+use gtk::{
+    glib::{self, ParamSpec},
+    prelude::*,
+    ResponseType,
+};
+
+glib::wrapper! {
+    pub struct MyAppWindow(ObjectSubclass<imp::MyAppWindow>)
+        @extends gtk::Widget, gtk::Window, gtk::ApplicationWindow;
+}
+
+#[gtk::template_callbacks]
+impl MyAppWindow {
+    pub fn new<P: glib::IsA<gtk::Application>>(app: &P) -> Self {
```

Comment:
```suggestion
    pub fn new<P: IsA<gtk::Application>>(app: &P) -> Self {
```
the trait is in prelude already